polkadot/node/service/src/lib.rs

```diff
@@ -1182,6 +1197,8 @@ pub fn new_full<OverseerGenerator: OverseerGen>(
 			sync: sync_service.clone(),
 			gossip_protocol_name: beefy_gossip_proto_name,
 			justifications_protocol_name,
+			notification_service: beefy_notification_service
```

Comment:
We could probably just check `if let Some(beefy_notification_service) = beefy_notification_service` instead of checking `if enable_beefy` here to avoid `expect`.